test/configs/alias.json

```diff
@@ -0,0 +1,8 @@
+{
+  "file": "alias",
```

Comment:
Please don't use config as json, this does not make it possible to test the js functions and other expressions